tests/lib/prepare-restore.sh

```diff
@@ -634,6 +634,11 @@ prepare_suite_each() {
         "$TESTSTOOLS"/cleanup-state pre-invariant
     fi
     tests.invariant check
+
+    SNAP_MOUNT_DIR="$(os.paths snap-mount-dir)"
+    if os.query is-focal || os.query is.fedora; then
+        inotifywait /root /var/lib/snapd /var/snap /home "$SNAP_MOUNT_DIR" -d -m -r -e CREATE -e DELETE --exclude "($PWD/task.yaml|$PWD/.*/.*.snap|$PROJECT_PATH/tests/lib/snaps/.*/.*.snap)" -o /tmp/fs.output
```

Comment:
Why track delete when we only check create events?

Reply:
Well, initially I tested differnet alternative, also tested files that could be deleted but at the end it was very complex to validate. I'll remove it. 
Thanks for the review